src/test/java/com/klarna/hiverunner/builder/HiveShellBaseTest.java

```diff
@@ -75,109 +85,109 @@ public void setupScriptsShouldBeExecutedAtStart() throws IOException {
 
     @Test(expected = IllegalStateException.class)
     public void setupScriptMayNotBeAddedAfterShellIsStarted() {
-        HiveShell shell = createHiveShell();
+        HiveShell shell = createHiveCliShell();
         shell.start();
         shell.addSetupScript("foo");
     }
 
     @Test(expected = IllegalArgumentException.class)
     public void invalidFilePathShouldThrowException() {
-        HiveShell shell = createHiveShell();
+        HiveShell shell = createHiveCliShell();
         shell.addSetupScripts(new File("foo"));
     }
 
 
     @Test(expected = IllegalStateException.class)
     public void setupScriptsMayNotBeAddedAfterShellIsStarted() throws IOException {
-        HiveShell shell = createHiveShell();
+        HiveShell shell = createHiveCliShell();
         shell.start();
         shell.addSetupScripts(tempFolder.newFile("foo"));
     }
 
     @Test
     public void executeScriptFile() throws IOException {
-      String hql = "use default;";
+      String hiveSql = "use default";
```

Comment:
I saw this earlier, why is it no longer necessary to put the ";" on the end? What happens if you do?